Src/VsVimShared/Implementation/Misc/FallbackKeyProcessor.cs

```diff
@@ -158,7 +177,7 @@ private bool TryConvertLetterToKeyInput(KeyEventArgs keyEventArgs, out KeyInput
             }
 
             var c = (char)('a' + (key - Key.A));
-            keyInput = KeyInputUtil.ApplyKeyModifiersToChar(c, keyModifiers);
+            keyInput = KeyInputUtil.ChangeKeyModifiersDangerous(KeyInputUtil.CharToKeyInput(c), keyModifiers);

```

Comment:
When ApplyKeyModifiersToChar, it alters the char and keyModifier so they are not picked up by the _fallbackCommandList. Here are some examples from the tests above:
````
Ctrl+a -> 1 '\u0001' with no KeyModifiers
Ctrl+m -> 13 '\r' with no KeyModifiers
Ctrl+r -> 18 '\u0012' with no KeyModifiers
````

When the ChangeKeyModifiersDangerous is used, the char and modifier are correctly passed along. Each of the tested keyboard shortcuts work.

Do you have any suggestions on how to get the best of both approaches?